src/components/ShareTrajectoryModal/style.css

```diff
@@ -0,0 +1,106 @@
+.upload-modal :global(.ant-modal-footer) {
+    padding: 12px 16px 20px;
+    background-color: var(--modal-content-bg);
+    max-height: 56px;
+}
+
+.upload-modal :global(.ant-modal-body) {
+    padding: 24px 24px 0px 24px;
+    font-size: 16px;
+    color: var(--dark);
+}
+
+.upload-modal :global(.ant-modal-header) {
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.upload-modal :global(.ant-modal-title) {
+    font-size: 14px;
+}
+
+.upload-modal :global(.ant-modal-close-x) {
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.upload-modal :global(.ant-checkbox-inner) {
+    background-color: transparent;
+    border: 1px solid var(--dark-three) !important;
+}
+
+input {
+    border-radius: 10px;
+}
+
+.blue-text {
+    color: var(--blue);
+    display: inline;
+}
+
+.body-text {
+    font-size: 14px;
```

Comment:
what's the default text size? I'm surprised how many font size settings there are in here. You should be able to use <h> tags to do some of the font adjustment so it's semantically styled

Reply:
ok, so i've added some global style rules for heading levels and removed local font-size styling, as well as changing html to be more semantic/accessible